Packs/CommonPlaybooks/Playbooks/playbook-User_Investigation_-_Generic.yml

```diff
@@ -0,0 +1,2791 @@
+      description: ''
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "58"
+      "yes":
+      - "30"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: QRadar.Search.Result
+                accessor: events
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1010,
+          "y": 1130
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "54":
+    id: "54"
+    taskid: 46fdfc73-7f2c-445f-860b-c239a93963e6
+    type: condition
+    task:
+      id: 46fdfc73-7f2c-445f-860b-c239a93963e6
+      version: -1
+      name: Does the result from Splunk exist?
+      description: ''
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "58"
+      "yes":
+      - "40"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: SplunkFailedLogonLogs
+                accessor: _raw
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -610,
+          "y": 1130
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "55":
+    id: "55"
+    taskid: 49054f44-d793-4785-8203-540913eee25c
+    type: condition
+    task:
+      id: 49054f44-d793-4785-8203-540913eee25c
+      version: -1
+      name: Does the result from Azure Log Analytics exist?
+      description: ''
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "58"
+      "yes":
+      - "26"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: AzureFailedLogonLogs.tables.rows.[0]
+                accessor: '[0]'
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -210,
+          "y": 1130
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "56":
+    id: "56"
+    taskid: 1481235c-ae5e-4914-81fd-f2bceaaef0f7
+    type: condition
+    task:
+      id: 1481235c-ae5e-4914-81fd-f2bceaaef0f7
+      version: -1
+      name: Does the result from Okta exist?
```

Comment:
Found results from Okta? (same for the tasks)

Reply:
Done